src/nvim/ex_cmds2.c

```diff
@@ -1847,19 +1852,21 @@ void ex_argdelete(exarg_T *eap)
     /* ":1,4argdel": Delete all arguments in the range. */
     if (eap->line2 > ARGCOUNT)
       eap->line2 = ARGCOUNT;
-    int n = eap->line2 - eap->line1 + 1;
+    long n = eap->line2 - eap->line1 + 1;
```

Comment:
you could, but then you need to add the casts `(int)n` below. (Though the growarray `ga_len` should really be `size_t`, but that would likely affect a lot of stuff so let's keep it `int` for this PR)